src/fixtures/base.js

```diff
@@ -13,3 +13,5 @@ export const test = base.extend({
         await use(app);
     },
 });
+
+export { expect };
```

Comment:
Are you imported expect from playwright and then exported it? Why did you do such a thing? Is there any reason?